.pre-commit-config.yaml

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+repos:
```

Comment:
I'll mention it here because I haven't fixed it yet in cuDF and this is a new file: the indentation of this file is ...aggressive. I'd prefer to see something more like 2 or 4 spaces instead of 8 spaces in some places and 2 or 4 in others. Maybe there's a standard YAML formatter that can be run once (no need to enforce the format with tooling, just get it into a normalized state).

Reply:
I usually add https://github.com/jumanjihouse/pre-commit-hook-yamlfmt to the pre-commit hooks for this reason.